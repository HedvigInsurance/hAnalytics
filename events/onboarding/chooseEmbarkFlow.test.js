const validateEvent = require("../../commons/validate-event.js")

test('validate chooseEmbarkFlow', () => {
    const event = validateEvent(__filename, {
        embarkStoryId: "embark_id"
    })

    expect(event).toMatchSnapshot()
});