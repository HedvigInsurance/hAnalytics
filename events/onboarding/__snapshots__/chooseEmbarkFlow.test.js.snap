// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate chooseEmbarkFlow 1`] = `
Object {
  "name": "onboarding_choose_embark_flow",
  "variables": Object {
    "embark_story_id": "embark_id",
  },
}
`;
